docs/application/native/guides/security/privilege.md

```diff
@@ -81,7 +81,23 @@ To get various privilege information:
                                                           &status);
     ```
 
-The above functions return a `privilege_info_error_e` enum value (in [mobile](../../api/mobile/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) and [wearable](../../api/wearable/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) applications) that indicates the value retrieval result. They also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.
+- Get the privilege information in a list form using the `privilege_info_get_privilege_info_list()` function:
+
+    ```
+    GList* privilege_name_list = NULL;
+    GList* privilege_info_list = NULL;
+    privilege_consumer_return_code_e return_result;
+
+    privilege_name_list = g_list_append(privilege_name_list, "http://tizen.org/privilege/call");
+    privilege_name_list = g_list_append(privilege_name_list, "http://tizen.org/privilege/contact.read");
+
+    int ret = privilege_info_get_privilege_info_list("ko_KR.UTF8",
+                                                     privilege_name_list,
+                                                     &privilege_info_list,
+                                                     &return_result);
+    ```
+
+The above functions return a `privilege_info_error_e` enum value (in [mobile](../../api/mobile/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) and [wearable](../../api/wearable/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) applications) that indicates the value retrieval result. These functions also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.
```

Comment:
@yl33 
Replace with: 

These functions return a `privilege_info_error_e` enum value in ([mobile](../../api/mobile/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) and [wearable](../../api/wearable/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) applications) that indicates the value of retrieval result. These functions also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.

Reply:
@nawab-samsung What is the reason of changing " (in mobile and wearable applications)" to " in (mobile and wearable applications)"?

Reply:
These functions return a privilege_info_error_e enum value in mobile and wearable applications, that indicates the value of retrieval result. These functions also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.

Reply:
@nawab-samsung As I requested, please stop breaking consistency. If you think removing () is better, please create a new PR to fix it for all files at once.

In this PR, please keep the original sentence.


Also, please add new comment rather than updating comment. Adding new comment sends mail to related people, but updating comment doesn't. As updating comment send no noti, it is easy to skip the updated opinion.

Reply:
@nawab-samsung

Reply:
@yl33  
yes, please update it with the text in your comment. 

These functions return `a privilege_info_error_e` enum value (in mobile and wearable applications), which indicates the value of retrieval result. These functions also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.

Reply:
@mijin85cho  sure thanks, will work on each page and ensure changes are consistent.

Reply:
@yl33  yes, kindly update with the text in your comment.

Reply:
@nawab-samsung  Done. Please check if I miss something.

Reply:
@yl33  thanks.
For maintaining consistency, accepting changes as per  https://github.com/Samsung/tizen-docs/pull/619#discussion_r265031242